From d0246418bb6b3d1e86d742a02332b1f03cab4d8a Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Sat, 4 Apr 2020 16:24:01 +0800
Subject: [PATCH 20/22] arch: arm64: not select IMX_GPCV2

There is an exclusive drvier(gpc-psci.c) from sdk kernel patch for
imx8 platform interrupt and power management, so remove the
IMX_GPCV2 selection to avoid confilt.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/Kconfig.platforms | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 8a2a70b36ff4..43918cf8a38b 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -165,7 +165,6 @@ config ARCH_MXC
 	bool "ARMv8 based NXP i.MX SoC family"
 	select ARM64_ERRATUM_843419
 	select ARM64_ERRATUM_845719 if COMPAT
-	select IMX_GPCV2
 	select IMX_GPCV2_PM_DOMAINS
 	select PM
 	select PM_GENERIC_DOMAINS
-- 
2.17.1

